components/permalink_view/permalink_view.jsx

```diff
@@ -103,44 +77,6 @@ export default class PermalinkView extends React.PureComponent {
             );
         }
 
-        return (
-            <div
-                id='app-content'
-                className='app__content'
-            >
-                <ChannelHeader
-                    channelId={channelId}
-                />
-                <PostView
-                    channelId={channelId}
-                    focusedPostId={match.params.postid}
-                />
-                <div
-                    id='archive-link-home'
-                >
-                    <Link
-                        to={'/' + teamName + '/channels/' + channelName}
-                        className='a11y__region'
-                        data-a11y-sort-order='2'
-                        innerRef={this.permalink}
-                    >
-                        {channelIsArchived &&
-                            <FormattedMarkdownMessage
-                                id='center_panel.permalink.archivedChannel'
-                                defaultMessage='You are viewing an **archived channel**. '
-                            />
-                        }
-                        <FormattedMessage
-                            id='center_panel.recent'
-                            defaultMessage='Click here to jump to recent messages. '
-                        />
-                        <LocalizedIcon
-                            className='fa fa-arrow-down'
-                            title={{id: t('center_panel.recent.icon'), defaultMessage: 'Jump to recent messages Icon'}}
-                        />
-                    </Link>
-                </div>
-            </div>
-        );
+        return null;
```

Comment:
we might want to add 
```js
console.warn(`DEPRECATED: ${window.location} is using deprecated permalink view`};
```
or some other way to inform of deprecation, not sure we have an stablished one.
so we know if something is trying to use this and we didn't notice

Reply:
`null` here is to make sure we don't show anything in the UI when url is about to change from `/pl/postId` to `/channel/postId`. As of now we will have the old one as the default url in the view for user. We internally redirect so pretty much any  permalink url will trigger the log. 

Let me know if it isn't clear from the comment